src/main/java/org/spongepowered/api/service/scheduler/Scheduler.java

```diff
@@ -26,60 +26,21 @@
 
 import com.google.common.base.Optional;
 
-import java.util.Collection;
+import java.util.Set;
 import java.util.UUID;
 
 /**
- * The base scheduler that schedules tasks.
+ * Represents a scheduler for running {@link Task}s.
  */
-
 public interface Scheduler {
```

Comment:
This should be actually called `SchedulerService` for consistency with the other services